apps/central-scan/backend/src/store.ts

```diff
@@ -1103,15 +1103,18 @@ export class Store {
     const loadedLayouts: BallotPageLayoutWithImage[] = [];
 
     for (const [pdf, layouts] of templates) {
+      let contestOffset = 0;
       for await (const { page, pageNumber } of pdfToImages(pdf, { scale: 2 })) {
         const ballotPageLayout = layouts[pageNumber - 1];
         loadedLayouts.push(
           await interpretTemplate({
             electionDefinition,
             imageData: page,
             metadata: ballotPageLayout.metadata,
+            contestOffset,
           })
         );
+        contestOffset += ballotPageLayout.contests.length;
       }
```

Comment:
Why are we using `interpretTemplate` in the scanner stores? We already have both the images and the layouts, why re-interpret??

Reply:
Yeah seems not necessary.